### PR TITLE
Add missing metrics validation and normalization to PWMetrics constructor

### PR DESCRIPTION
**Pull request by MickeyKay**
Addresses https://github.com/paulirish/pwmetrics/issues/36

Also might be worth updating the readme demo configs to simply omit the `>=`. Is there a reason why those are included if they're just later stripped?

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -23,6 +23,9 @@ class PWMetrics {
     this.opts.disableCpuThrottling = this.opts.disableCpuThrottling || false;
     this.metrics = opts.metrics;
```
Comment (denar90) [inline lib/index.js]: `this.metrics` is redundant, it's already removed in `sheets` branch. But your point is really good. Can you rebase on `sheets-finalizations` branch and make your changes there. It's kinda new version branch where a lot of changes were done. I hope soon @paulirish accept it and we release all this stuff.

Comment (paulirish) [inline lib/index.js]: master is now updated, so you'll need to rebase to that. :)

**Comment by MickeyKay** [comment]
Woah, I just reset this branch and forced a push on my end and suddenly it's saying this PR is merged. Never seen that before. I will refile.

File: lib/expectations.js [hunk 2]
```diff
@@ -31,9 +31,6 @@ function normalizeMetrics(metrics) {
 }
 
 function checkExpectations(metricsData, expectationMetrics) {
-  validateMetrics(expectationMetrics);
```
Comment (MickeyKay) [inline lib/expectations.js]: We can now remove this since it being called right off the bat in our constructor.

**Comment by MickeyKay** [comment]
@denar90 @paulirish Updated, however I ran into some weird rebasing issues that I wasn't able to cleanly resolve, so I started fresh and included the minimum changes required based off the most recent master branch.

**Comment by denar90** [comment]
Merging this. We can improve it in master if needed.

**Pull request merged**